syft/pkg/cataloger/cataloger.go

```diff
@@ -45,7 +45,7 @@ func ImageCatalogers(cfg Config) []pkg.Cataloger {
 		apkdb.NewApkdbCataloger(),
 		binary.NewCataloger(),
 		deb.NewDpkgdbCataloger(),
-		dotnet.NewDotnetDepsCataloger(),
+		dotnet.NewDotnetPortableExecutableCalaloger(),
```

Comment:
this can also be added to the DirectoryCatalogers as well

Reply:
Sounds good!